src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -46,29 +46,39 @@ class FireCloudServiceActor extends HttpServiceActor {
   def receive = runRoute(
     logRequests {
       swaggerCorsService ~
-      // The "service" path prefix is never visible to this server in production because it is
-      // transparently proxied. We check for it here so the redirects work when visiting this
-      // server directly during local development.
-      pathPrefix("service") {
-        swaggerUiService ~
-          testNihService ~
-          oAuthService.routes ~
-          userService.routes ~
-          pathPrefix("api") {
-            routes
-          }
+      swaggerUiService ~
+      testNihService ~
+      oAuthService.routes ~
+      userService.routes ~
+      nihSyncService.routes ~
+      pathPrefix("api") {
+        routes
       } ~
-        swaggerUiService ~
-        testNihService ~
-        oAuthService.routes ~
-        userService.routes ~
-        nihSyncService.routes ~
-        pathPrefix("api") {
-          routes
-        }
+      downloadEntitiesService
     }
   )
 
+  val downloadEntitiesService = {
+    get {
+      path("workspaces" / Segment / Segment / "entities" / Segment / "tsv") {
+        (workspaceNamespace, workspaceName, entityType) =>
+          val baseRawlsEntitiesUrl = FireCloudConfig.Rawls.entityPathFromWorkspace(
+            workspaceNamespace, workspaceName)
+          val entityTypeUrl = encodeUri(baseRawlsEntitiesUrl + "/" + entityType)
+          get { requestContext =>
+            val builder = new RequestBuilder(HttpMethods.GET)
+            val externalRequest = builder(encodeUri(entityTypeUrl), requestContext.request.entity)
+            val authHeader = HttpHeaders.Authorization(OAuth2BearerToken("ya29.hALCXZExBomdA6FgKk8ekCBxJ6JjA6t8pyCFiXhUB9tSFpLz7RSRg_mXJ3ds3UeJYyRcmtlP-1Ife7u7"))
+            val headers = requestContext.request.headers :+ authHeader
+            externalHttpPerRequest(
+              requestContext.withRequestMapped(_.withHeaders(headers)),
+              externalRequest
+            )
+          }
+      }
+    }
+  }
```

Comment:
Why is this route here? This endpoint is already handled in EntityService in a very different way.